subsys/bluetooth/host/monitor.c

```diff
@@ -25,6 +25,8 @@
 
 #include <bluetooth/buf.h>
 
+#include <SEGGER_RTT.h>
```

Comment:
Is it safe to unconditionally include it like this, considering that this comes from an external module and not the main tree? Perhaps make it depend on CONFIG_BT_DEBUG_MONITOR_RTT?